changelog.txt

```diff
@@ -28,7 +28,10 @@ Template for new versions:
 
 ## New Tools
 - `sync-windmills`: synchronize or randomize movement of active windmills
-- `trackstop`: new overlay to allow changing track stop dump direction and friction and roller direction and speed after construction
+- `trackstop`: provides 3 new overlays:
+    - trackstop: allow changing track stop dump direction and friction
+    - rollers: allow changing roller direction and speed
+    - reorderstops: reorder stops in hauling routes
```

Comment:
this kind of formatting doesn't translate well to the final rendered changelog. each line should be independent. If you start each of them with ``` `trackstop`: ``` then they will get combined into a list similar to your intention here